Swerve/SwerveModules/SwerveModule.java

```diff
@@ -26,11 +27,21 @@
 import edu.wpi.first.wpilibj.smartdashboard.SmartDashboard;
 import edu.wpi.first.wpilibj.sysid.SysIdRoutineLog;
 
-public abstract class SwerveModule {
+import edu.wpi.first.wpilibj.Alert;
+import edu.wpi.first.wpilibj.Alert.AlertType;
+import org.littletonrobotics.junction.Logger;
+
+public abstract class SwerveModule implements SwerveModuleIO{
```

Comment:
Are there any outputs we should be logging? Motor encoder positions, current swerve module states, etc ...